### ISSUE TITLE
Universal Button Text: Button text Contrast - 1.4.3 Contrast (Minimum) 

### ISSUE THREAD

**Issue by syarwick** [stephenravenscraft/loyola-template-4.0#63]
The maroon buttons on the gold background have black text, which does not meet a sufficient contrast ratio of 4.5:1.  Update the font color to white.

**Comment by syarwick** [comment]
Hi Steve, 

the wording here is kinda confusing, my bad. The buttons are maroon, the text is black,a nd the background (behind the buttons) is gold. I just meant the black text on maroon buttons only appears on the gold background. 

![Image](https://github.com/user-attachments/assets/41e3aa6f-41d1-4962-b713-35eb90c232bd)

**Comment by syarwick** [comment]
In this case, it's the third set of buttons here
